.circleci/config.yml

```diff
@@ -93,6 +93,32 @@ commands:
             pip install --upgrade setuptools
             pip install sphinx sphinx_rtd_theme
 
+  configure-build-system:
+    description: "Modifies the SConstruct file to use grackle or not."
+    parameters:
+      usegrackle:
+        type: boolean
+        default: true
+    steps:
+      - run:
+          name: "Modify the use_grackle parameter in SConstruct."
+          command: |
+            # convert boolean parameter to an env var storing 0 or 1
+            USE_GRACKLE=$(( 0 <<# parameters.usegrackle >> + 1 <</ parameters.usegrackle >> ))
+
+            source $BASH_ENV
+            # to help make sure that this doesn't fall through the cracks if
+            # parts of the build system change, check the validity of the default
+            # value of the use_grackle parameter
+            DFLT_GRACKLE_VAL=$(sed -n 's/^use_grackle *= *//p' SConstruct)
+            if [[ "$DFLT_GRACKLE_VAL" != "0" && "$DFLT_GRACKLE_VAL" != "1" ]];
```

Comment:
Just to a note to everyone else that this is a github rendering issue. The actual code reads
`if [[ "$DFLT_GRACKLE_VAL" != "0" && "$DFLT_GRACKLE_VAL" != "1" ]];`